From 12ac8441f75bab2a256278926fe32450a311af1b Mon Sep 17 00:00:00 2001
From: Nicolas Dufresne <nicolas.dufresne@collabora.com>
Date: Mon, 4 Jul 2016 15:07:48 -0400
Subject: build: Add required define for Windows

This will also add it to the .pc file as WEBRTC_WIN leaks into the
public interface and undefined __STRICT_ANSI__ so M_PI is available.

https://bugs.freedesktop.org/show_bug.cgi?id=96754

diff --git a/configure.ac b/configure.ac
index f34be72..ccaa758 100644
--- a/configure.ac
+++ b/configure.ac
@@ -65,8 +65,11 @@ AS_CASE(["${host}"],
              [OS_FLAGS="-DWEBRTC_MAC -DWEBRTC_IOS"],
              [OS_FLAGS="-DWEBRTC_MAC"])
          PLATFORM_CFLAGS="-DWEBRTC_POSIX"
+        ],
+    [*-mingw32*],
+        [
+         PLATFORM_CFLAGS="-DWEBRTC_WIN -U__STRICT_ANSI__"
         ]
-    # FIXME: Add Windows support
 )
 AC_SUBST(PLATFORM_CFLAGS)
 
-- 
cgit v0.10.2

